### DISCUSSION TITLE
Let's establish default base lines for the distros

### DISCUSSION THREAD

**Discussion by rkwesk** [ltsp/ltsp#172] [Q&A]
It seems to me that Ubuntu is easier for this since almost all its users start with a desktop live and choose a standard install rather than the minimum install. The choice of desktop is the variable.

Debian (imho) is different because its users probably choose a variety of installs. Also, I'm guessing that a Debian install may not be a clean start but rather a user already may be using it as a server and now decides to add ltsp to it.

Why not make a list of desirable (not dependencies exactly) pkgs so there are a few established scenarios to work with.

Perhaps Fedora may also need this?

What do others think about this? I suspect that certain scenarios will arise from contributions from different users.

Richard

**Comment by alkisg** [comment]
I think that sysadmins that don't want a desktop environment on the server, and will use a VM or chroot, can keep it minimal and just follow https://github.com/ltsp/ltsp/wiki/installation.

Those that want a desktop environment can select one from the Debian (or any other distro) installer, and then again follow https://github.com/ltsp/ltsp/wiki/installation.

![](https://i.stack.imgur.com/tJQ3w.png)

**Comment by rkwesk** [comment]
Returning to this issue, or should it warrant a separate new issue?

Am I correct in my understanding that the ltsp19 code indeed fits Ubuntu with resolveconf and networkd-dispatcher installed as well as Debian without either of these (as I have had it installed?) I ask because in my limited understanding elswhere I had created race conditions and never saw the proper way to set things up.

Richard

**Comment by alkisg** [comment]
My basic idea there is "the [LTSP installation instructions](https://github.com/ltsp/ltsp/wiki/installation) should work out of the box in any distribution, as long as the sysadmin is using the distro defaults".

For example, current Ubuntu by default is NOT using resolvconf.
Current Debian IS using resolvconf.
Both of those cases should work out of the box by just following the installation instructions.
If they don't, bugs should be filed and they should be addressed.

On the other hand, if a sysadmin doesn't want to follow the defaults, e.g. if he wants to use tftpd-hpa in Ubuntu instead of dnsmasq, he should be prepared to edit its configuration file and point it to /srv/tftp instead of /var/lib/tftpboot.

So the "proper way to set things up" is:
- Follow the installation instructions of your distribution,
- And then follow the LTSP installation instructions

If these don't work out of the box, file bugs. :)
